open-craft/edx-platform
# [BB-1453] Pre-install numpy before installing matplotlib

This is because, `matplotlib` tries to download the latest version
of `numpy` (1.17.0rc1), which is not compatible with python 2.7 any more, during its installation and causes the provisioning to fail because of the error raised by `numpy`.

**Testing instructions**:
* Create a new instance using the PR branch and the related configuration PR's (https://github.com/open-craft/configuration/pull/105) branch.
* Verify that the provisioning doesn't fail in the step where the edx sandbox requirements are installed and confirm that all the requirements in `requirements/edx-sandbox/pre.txt` and `requirements/edx-sandbox/base.txt` are installed in the sandbox virtualenv environment.
